Starwar/Assets/Scripts/Player Control/Player_Control.cs

```diff
@@ -48,14 +53,22 @@ private void FixedUpdate()
         if (Input.GetButton("Fire1"))
         {
             Debug.Log("Fire bullet");
+            soundController.playLaser();
```

Comment:
no need here, you played it on MachineGunManager.
just change MachineGun to MachineGunManager.